www/docs/customization/milestone.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Milestone
+---
+
+GoReleaser can close GitHub/GitLab/Gitea milestones after successfully
+publishing all artifacts.
+
+Let's see what can be customized in the `milestones` section:
+
+```yaml
+# .goreleaser.yml
+milestones:
+  # You can have multiple milestone configs
+  -
+    # GitHub repository for the milestone
+    # Default is extracted from the origin remote URL
+    github:
+      owner: user
+      name: repo
+
+    # GitLab project for the milestone
+    # Default is extracted from the origin remote URL
+    gitlab:
+      owner: user
+      name: repo
+
+    # Gitea repository for the milestone
+    # Default is extracted from the origin remote URL
+    gitea:
+      owner: user
+      name: repo
+
+    # Whether to close the milestone
+    # Default is true
```

Comment:
Maybe the default should be false? Otherwise people might just upgrade goreleaser and get their milestones closed without understanding why 🤔

Reply:
Oh ha sorry, the default was false since the configuration is boolean and it wasn't defaulted. Fixing the documentation page. 👍

Reply:
Updated in https://github.com/goreleaser/goreleaser/pull/1657/commits/5aea6dd9fed3c81f9878bae771456b73722ba4a7 👍